Formula/devtodo.rb

```diff
@@ -3,6 +3,7 @@ class Devtodo < Formula
   homepage "https://swapoff.org/devtodo.html"
   url "https://swapoff.org/files/devtodo/devtodo-0.1.20.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/alecthomas/devtodo2/archive/refs/tags/2.1.tar.gz"
```

Reply:
Note that more changes to the formula are required to build v2, as it is written in Go, while v1 is written in C++.